pipelines/_ingredients/ConversionsCalibrateImage.yaml

```diff
@@ -0,0 +1,20 @@
+# TODO DM-40388: This file will be merged in to Conversions.yaml, once the new pipeline is vetted.
+# Add-on pipeline designed to be added to ApVerify or ApPipe for QA purposes.
+#
+# This pipeline depends on an external pipeline for the coaddName and fakesType
+# pipeline parameters.
+
+imports:
+  - location: $AP_VERIFY_DIR/pipelines/_ingredients/Conversions.yaml
```

Comment:
Just to double-check, did you mean for the new file to depend on the old one? It will make DM-40388 harder than if the new file is standalone and the old one imports and tweaks it.

Reply:
Yes: my intention here is to try not to break any existing pipelines. I'm having enough trouble testing and debugging these new ones (the middleware output for pipeline validation continues to be insufficient for finding *where* the failure is), and I don't want to have to re-validate the old pipelines.